### ISSUE TITLE
Support Double Click to Choose on Mobile

### ISSUE THREAD

**Issue by jpdevries** [jpdevries/eureka#98]
React has `onDoubleClick` but it doesn't work on phones. At least not yet. This support is much needed, especially in large folders, in the two views that don't have tap-to-open context menus
